packages/instant-apps-components/src/instant-apps-sign-in.html

```diff
@@ -39,6 +43,8 @@
         id="instant-apps-header"
         slot="header"
         title-text="Instant Apps: Header (Shared Theme)"
+        logo-image="https://www.esri.com/content/dam/esrisites/en-us/common/icons/product-logos/arcgis-instant-apps-64.svg"
```

Comment:
Is this hardcoded value needed? Can we include the logo locally so it'll work in offline enterprise sites?

Reply:
This is just the test file